README.md

```diff
@@ -33,32 +33,32 @@ print(line.url)
 
 
 ## Getting started
-To get started with novem you'll have to register an account, currently this
-can be done by reaching out to the novem developers on hello@novem.no.
+To get started with novem you will have to register an account. Please
+[reach out](mailto:hello@novem.no) to us!
```

Comment:
Update to novem.io

Reply:
We can do that after the email switch-over.